website/static/js/share/utils.js

```diff
@@ -1,4 +1,5 @@
 var m = require('mithril');
+var $osf = require('js/osfHelpers');
 var History = require('exports?History!history');
 
 var callbacks = [];
```

Comment:
Returning a promise in `loadMore` may also get rid of this "global" callbacks variable.